README.md

```diff
@@ -1,6 +1,6 @@
 [![Build Status](https://travis-ci.org/CSCfi/ansible-role-dhcp-kickstart.svg?branch=master)](https://travis-ci.org/CSCfi/ansible-role-dhcp-kickstart)
 
-Spacewalk proxy  Ansible role
+DHCP/PXE privisioning Ansible role
```

Comment:
A "couldn't find any errors or anything smart to say but want to review at least something" comment:

`s/privisioning/provisioning/`

Reply:
Awww shucks, didn't refresh page as this got merged, nevermind

Reply:
Ran aspell -c README.md and found another typo. Pushed to master :s